fixtures/clay-paky/spheriscan.json

```diff
@@ -0,0 +1,929 @@
+          "index": 1,
+          "comment": "Clouds",
+          "shakeSpeedStart": "slow",
+          "shakeSpeedEnd": "fast"
+        },
+        {
+          "dmxRange": [151, 171],
+          "type": "GoboIndex",
+          "index": 2,
+          "comment": "Eccentric Circle",
+          "shakeSpeedStart": "slow",
+          "shakeSpeedEnd": "fast"
+        },
+        {
+          "dmxRange": [172, 192],
+          "type": "GoboIndex",
+          "index": 3,
+          "comment": "Multiple Cones",
+          "shakeSpeedStart": "slow",
+          "shakeSpeedEnd": "fast"
+        },
+        {
+          "dmxRange": [193, 213],
+          "type": "GoboIndex",
+          "index": 4,
+          "comment": "Fat Line",
+          "shakeSpeedStart": "slow",
+          "shakeSpeedEnd": "fast"
+        },
+        {
+          "dmxRange": [214, 234],
+          "type": "GoboIndex",
+          "index": 5,
+          "comment": "Half Circle",
+          "shakeSpeedStart": "slow",
+          "shakeSpeedEnd": "fast"
+        },
+        {
+          "dmxRange": [235, 255],
+          "type": "GoboIndex",
+          "index": 6,
+          "comment": "Linear Dots",
+          "shakeSpeedStart": "slow",
+          "shakeSpeedEnd": "fast"
+        }
+      ]
+    },
+    "Gobo Stencil Rotation": {
+      "fineChannelAliases": ["Gobo Stencil Rotation fine"],
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 125],
+          "type": "GoboStencilRotation",
+          "angleStart": "0deg",
+          "angleEnd": "540deg"
+        },
+        {
+          "dmxRange": [126, 190],
+          "type": "GoboStencilRotation",
+          "speedStart": "180rpm",
+          "speedEnd": "0.36rpm"
+        },
+        {
+          "dmxRange": [191, 192],
+          "type": "GoboStencilRotation",
+          "speed": "stop"
+        },
+        {
+          "dmxRange": [193, 255],
+          "type": "GoboStencilRotation",
+          "speedStart": "-0.36rpm",
+          "speedEnd": "-180rpm"
+        }
+      ]
+    },
+    "Prism insertion": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 127],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [128, 255],
+          "type": "Prism",
+          "comment": "8-facet"
+        }
+      ]
+    },
+    "Prism Rotation": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 127],
+          "type": "PrismRotation",
+          "angleStart": "0deg",
+          "angleEnd": "540deg"
+        },
+        {
+          "dmxRange": [128, 190],
+          "type": "PrismRotation",
+          "speedStart": "80rpm",
+          "speedEnd": "0.05rpm"
+        },
+        {
+          "dmxRange": [191, 192],
+          "type": "PrismRotation",
+          "speed": "stop"
+        },
+        {
+          "dmxRange": [193, 255],
+          "type": "PrismRotation",
+          "speedStart": "0.05rpm",
```

Comment:
This should probably be a `-0.05rpm`

Reply:
Correct. We probably have to improve the UI for this case, as it isn't clear that negative speed means counterclockwise direction.